<?xml version="1.0"?>
<launch>

  <arg name="robot_name" default="changeme"/>
  <arg name="config" default="$(find power_monitor)/config/microchip_mcp3424.yaml"/>

  <node name="power_monitor_node" pkg="power_monitor" type="power_monitor_node" output="screen">
      <rosparam command="load" file="$(arg config)"/>
      <param name="frame_id" value="$(arg robot_name)/power_minitor"/>
  </node>

</launch>